ci/arm/gitlab-ci.yml

```diff
@@ -0,0 +1,10 @@
+linux-arm64:
```

Comment:
What base image do you want?
(please include an `image:` field)

Reply:
I was just doing a shell job, that way ccache can be used without a bunch of hacks, and it is just much simpler, as Zig has no need for root.

Feel free to do it your own way.